src/streaming/constants/Constants.js

```diff
@@ -260,6 +260,7 @@ export default {
         START: 's',
         PLAYING: 'p',
         PAUSED : 'a',
+        WAITING: 'w',
         SEEKING: 'k',
         REBUFFERING: 'r',
         FATAL_ERROR: 'f',
```

Comment:
The CMCD_REPORTING_EVENTS aren't the same as the player states. For example, the key `e` in the reporting events is `error`, while it is `ended` in the player state.

We should create two separate constants, the CMCD_REPORTING_EVENTS and the CMCD_PLAYER_STATES